cocos/renderer/CCTextureCache.cpp

```diff
@@ -102,6 +102,12 @@ void TextureCache::addImageAsync(const std::string &path, const std::function<vo
 
     std::string fullpath = FileUtils::getInstance()->fullPathForFilename(path);
 
+    // check if file exists
+    if ( fullpath.empty() || ! FileUtils::getInstance()->isFileExist( fullpath ) ) {
+        callback(nullptr);
+        return;
+    }
+    
```

Comment:
If the image already in texture, it's allow to loaded. So the code need to move the comment  after the textures find.